config.toml

```diff
@@ -27,3 +27,29 @@ permissions-bools = [
     "dev-desktop",
     "sync-team-confirmation",
 ]
+
+# GitHub accounts that are allowed to stay in the orgs,
+# even if they may not be members of any team.
+allowed-org-members = [
```

Comment:
nit: `allowed` makes it sound like only these org members maybe a part of the org :D Which of course doesn't make sense, and the comment does clarify it. But I don't have a great alternative. Maybe just `special-org-members`? But it doesn't really matter.

Reply:
Special also doesn't clarify much, maybe call it `dont-remove-from-orgs = []`?